client/js/widgets/seat.js

```diff
@@ -42,6 +42,15 @@ class Seat extends Widget {
     this.updateLinkedWidgets(true);
   }
 
+  childrenTarget() {
```

Comment:
Any reason not to override children() like holder.js does?

Reply:
children() is actually used in some other places and I'm afraid it might break something if I change the seats' children().
I might be wrong; this can be an unnecessary extra function, but I feel safer this way.

This is one of the things I'm most unsure about this PR but at the same time it doesn't feel too problematic